zerver/tests/test_decorators.py

```diff
@@ -1667,7 +1667,7 @@ def test_require_non_guest_user_decorator(self) -> None:
         guest_user = self.example_user('polonius')
         self.login_user(guest_user)
         result = self.common_subscribe_to_streams(guest_user, ["Denmark"], allow_fail=True)
-        self.assert_json_error(result, "Not allowed for guest users")
+        self.assert_json_error(result, "Unable to access stream (Denmark).")
```

Comment:
This message is bad for debugging purposes. We should add new error message specific to the case being handled.